Constitution.txt

```diff
@@ -1,490 +1,264 @@
+2: Every Bill which shall provide for the funding of actions of the Federal Government of these United States prescribed by this Constitution which has been presented first to the House of Representatives, received the approval and signature of two-thirds of their number, presented to the Senate, and received the approval and signature of two-thirds of their number, shall be presented to the President for his approval. If he should approve the Bill by signature, it shall become Law; if not, or if he shall not provide his approval within 14 calendar days of the Senate's approval, it shall be returned to the House of Representatives, and if three-fourths of the Representatives approve, it shall become law. Bills passed into law shall retain force of law for a period to be defined in the Bill, but not to exceed six years.
 
-3: Every Order, Resolution, or Vote to which the Concurrence of the Senate and House of Representatives may be necessary (except on a question of Adjournment) shall be presented to the President of the United States; and before the Same shall take Effect, shall be approved by him, or being disapproved by him, shall be repassed by two thirds of the Senate and House of Representatives, according to the Rules and Limitations prescribed in the Case of a Bill.
+3: Every other Bill which has been presented to the House of Representatives and the Senate, and has been approved and signed by all Representatives and Senators from several of the States, shall be sent to the Chief Executive of each of those States. If he should approve the Bill, it shall be considered approved by his State; if not, he shall send it to the Legislature of his State. If, after consideration, the Legislature of the State approves and signs the Bill in keeping with that State's Constitution, it shall be considered approved by the State. When at least two States have approved the Bill, it shall become law in those states which have so approved it, for a period not to exceed six years from first approval.
 
 Section 8
-1: The Congress shall have Power To lay and collect Taxes, Duties, Imposts and Excises, to pay the Debts and provide for the common Defence and general Welfare of the United States; but all Duties, Imposts and Excises shall be uniform throughout the United States;
+The Congress shall have Power:
 
-2: To borrow Money on the credit of the United States;
+1:  To lay and collect Taxes, Duties, Imposts, and Excises, on commerce among the several States and with foreign nations; to pay the Debts and provide for the common Defense of these United States; but all Duties, Imposts and Excises shall be uniform throughout these United States; and no Tax, Duty, Impost, or Excise shall apply to transactions occurring wholly within one State, nor shall any resident of any State be compelled by law to participate in a transaction other than the payment of such Taxes, Duties, Imposts, and Excises.
```

Comment:
Along with a formatting change that causes number offsets, I add here a power to lay tariffs on the borders between the States. I will expound upon one of its purposes later in this document, but I'll list other reasons and effects here.

- A tariff, being a tax, will have the effect of reducing the trade across state boundaries as compared to free market conditions. This will have the economic effect of reducing overall wealth, as all impositions against the market do, but will purchase for that cost a systemic incentive for people to buy local goods within their States. This will encourage local economies to be well-diversified, and help to preserve the economic independence of a given State.
- As you will later see, I have removed the authority of the Congress to lay direct taxes against income. This limits the Congress to raising funds either by external tariff, internal tariff, or tax against the States directly. I expect that the majority of revenues would come from external tariff, with internal tariffs kept low to encourage prosperity, and direct taxes against States avoided except in budgetary emergencies.